Client/Frontend/Library/HistoryPanel/HistoryPanel.swift

```diff
@@ -761,8 +761,7 @@ extension HistoryPanel {
         updatePanelState(newState: .history(state: .mainView))
 
         TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .deleteHistory)
-        // TODO: Yoana remove notification and handle directly
```

Comment:
@yoanarios 😆 Seems this was missed at some point haha. Thanks for fixing that @FilippoZazzeroni

Reply:
Ahhaha indeed. You're welcome ☺️